packages/forms-web-app/src/views/manage-appeals/appeal-details.njk

```diff
@@ -22,10 +22,6 @@
 </style>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      {{ govukBackLink({
-        text: "Back to appeals",
```

Comment:
should be ok to remove this but if using default backlink functionality, it will take user back to last page they visited which might not always be the appeals page. Because the text of the backlink in this case is specific, I think that's why it was hardcoded.

Reply:
ah ok, I'll see if I can get that to work somehow through the template as the current solution appears too low down as it's within the main.
If I can't figure anything out, I'll fix the link and leave positioning as is for now